Makefile

```diff
@@ -40,6 +40,17 @@ EXT_OPTIONS_MENU ?= 1
 TARGET_WEB ?= 0
 # Specify the target you are building for, 0 means native
 TARGET_ARCH ?= native
+
+ifeq ($(CROSS),i686-w64-mingw32.static-)
+  ifeq ($(CROSS),x86_64-w64-mingw32.static-)
```

Comment:
This ifeq seems redundant. It will only enter if CROSS is set to i686, which means it won't be set to x86_64. Also, it seems like you're setting the target architecture to i386pe for an x86-64 build. Why's that?

Reply:
ld complains about having the emulation type be i386pep if this isn't set, which is why this came into play. Will fix that minor issue

Reply:
Issue's been fixed as of latest commit.